src/components/Authentication/Kratos/KratosInput.tsx

```diff
@@ -40,8 +39,8 @@ export const KratosInput: FC<KratosInputProps> = ({ node, autoCapitalize, autoCo
       ...InputProps,
       endAdornment: (
         <InputAdornment position="end">
-          <IconButton onClick={() => setInputType(inputType === 'password' ? 'text' : 'password')} size="large">
-            <Icon component={inputType === 'password' ? Eye : EyeSlash} color="inherit" size={'xs'} />
+          <IconButton onClick={() => setInputType(inputType === 'password' ? 'text' : 'password')} size="small">
+            {inputType === 'password' ? <VisibilityIcon /> : <VisibilityOffIcon />}
```

Comment:
Makes sense to extract `inputType === 'password'` to a variable like `isPasswordVisible`. It's not for optimization or DRY but I see it as an improvement to readability: it makes clear that actually it's not the field type that changes, but the visibility state of the password input.